ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -1327,6 +1327,19 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-azure-ipi-private
+- as: azure-ipi-private-spec-net-type-spec-perm-f28
```

Comment:
Can we add a pair of jobs into multi-arch ci, one is for arm + e2e, another for amd + destructive.

Reply:
updated